### PR TITLE
feat: dry run analytics

### PR DESCRIPTION
**Pull request by robertbrada**
Tracks if the user's attempt to check seed was successful or not

### CODE REVIEW

File: packages/suite/src/views/recovery/index.tsx [hunk 1]
```diff
@@ -305,7 +306,17 @@ const Recovery = ({
                         </StyledP>
                         <StyledImage image="UNI_SUCCESS" />
                         <Buttons>
-                            <CloseButton onClick={() => closeModalApp()} />
+                            <CloseButton
+                                onClick={() => {
+                                    closeModalApp();
```
Comment (szymonlesisz) [inline packages/suite/src/views/recovery/index.tsx:320]: wouldnt be better to create one function for that, like `onSetRecoveryType` (line 128) with a success: boolean parameter and use it in both cases, here and below?

**Comment by mroz22** [comment]
And what about reporting it from here? 
https://github.com/trezor/trezor-suite/blob/1da73bf5c45ed462662a4b3f12b10993625b9999/packages/suite/src/actions/recovery/recoveryActions.ts#L70

I mean - the closer to the source of truth, the better results.

**Comment by robertbrada** [comment]
> And what about reporting it from here?
> https://github.com/trezor/trezor-suite/blob/1da73bf5c45ed462662a4b3f12b10993625b9999/packages/suite/src/actions/recovery/recoveryActions.ts#L70
> 
> I mean - the closer to the source of truth, the better results.

Hmm, actually I see the analytics I want to track is implemented there already as **check-seed/error** event type, just didn't see it before haha. So I will just update the Notion documentation and this PR can be deleted!

**Comment by mroz22** [comment]
Ah I didnt notice that :D lol, good

**Pull request closed without merging**